src/multiselect/embedded.tsx

```diff
@@ -77,12 +74,7 @@ const EmbeddedMultiselect = React.forwardRef(
     const status = multiselectProps.dropdownStatus;
 
     return (
-      <div
-        role="group"
```

Comment:
The wrapper does not need to be a semantic element, the ARIA label is already attached to the list element.